docs/internals/deprecation.txt

```diff
@@ -20,6 +20,8 @@ details on these changes.
   ``password_reset_confirm()``, and ``password_reset_complete()`` will be
   removed.
 
+* ``contrib.auth.views.logout_then_login()``'s ``extra_context`` parameter will be removed
```

Comment:
Add a period and wrap lines at 79 characters. Also you need to use a "right quote" for `'s`, otherwise Sphinx renders it backwards.